content/ecosystem/members.json

```diff
@@ -1,54 +1,128 @@
 [
   {
-    "name": "bosonic-qiskit",
-    "url": "https://github.com/C2QA/bosonic-qiskit",
-    "description": "NQI C2QA project to simulate hybrid boson-qubit systems within Qiskit.",
-    "licence": "BSD 2-Clause Simplified or FreeBSD license",
+    "alternatives": null,
```

Comment:
As this is not related to your PR, please ensure that it's not included.

Thanks @AanshSavla

Reply:
I have removed the members.json changes.